raiden_contracts/constants.py

```diff
@@ -8,7 +8,7 @@
 from raiden_contracts.utils.type_aliases import ChainID
 
 # Do not change this, this is handled by bumpversion with .bumpversion_contracts.cfg
-CONTRACTS_VERSION = "0.25.2"
+CONTRACTS_VERSION = "0.33.0"
```

Comment:
Just using `0.33` would make it more obvious that this is not affected by patch releases. There might be downsides to that approach, but I don't see any, right now.

Reply:
I'm using `semver.compare()` on these strings.  Perhaps I need to find another way to compare versions.  Let me keep the `.0` at the end for now.